src/LtiDeployment.php

```diff
@@ -2,21 +2,26 @@
 
 namespace Packback\Lti1p3;
 
-class LtiDeployment
+use Packback\Lti1p3\Interfaces\ILtiDeployment;
+
+class LtiDeployment implements ILtiDeployment
 {
-    private $deployment_id;
+    public function __construct(
+        private $deployment_id
+    ) {
```

Comment:
Hmm, the existing code allows `LtiDeployment` to be instantiated without specifying a `$deployment_id`; do we want to require users to specify one upon instantiating? If so, then it may be worth listing this in `UPGRADES.md`. Also, `$deployment_id` isn't typed; should it be of type `string | int`?

Reply:
I did some poking around, and I think `string|int` might work, but given the uncertainty of how various LMSes will send this data over, I think it's safer to leave this untyped